salt/modules/zcbuildout.py

```diff
@@ -1032,14 +1032,20 @@ def _check_onlyif_unless(onlyif, unless, directory, runas=None, env=()):
                 if not onlyif:
                     _valid(status, 'onlyif execution failed')
             elif isinstance(onlyif, string_types):
-                if retcode(onlyif, cwd=directory, runas=runas, env=env) != 0:
+                if retcode(
```

Comment:
np, but the default value isnt changed after all if you reread well, im going to split into two and rebase this request to include only one change (state i think) and open the other counter part.